pkg/rewards/claim.go

```diff
@@ -187,7 +313,15 @@ func Claim(cCtx *cli.Context, p utils.Prompter) error {
 		}
 
 		logger.Infof("Broadcasting claim transaction...")
-		receipt, err := eLWriter.ProcessClaim(ctx, elClaim, config.RecipientAddress, true)
+
+		var receipt *types.Receipt
+
+		if len(*elClaims) > 0 {
+			receipt, err = eLWriter.ProcessClaims(ctx, *elClaims, config.RecipientAddress, true)
+		} else {
+			receipt, err = eLWriter.ProcessClaim(ctx, (*elClaims)[0], config.RecipientAddress, true)
+		}
```

Comment:
A bit confused by this. If the size is not greater than 0, you are passing the 0th index value?

Reply:
The idea is here is if multipe claims have been passed in to call to call the batch `ProccessClaims` and if it is just 1 to call `ProcessClaim`

Reply:
That makes sense - I think I'm confused by the `> 0`. Based on what you described, I was thinking this would be `> 1`.

Reply:
yep good catch. this is is a bug!